.circleci/config.yml

```diff
@@ -382,10 +382,10 @@ jobs:
             pnpm lint:check || echo "export LINT_STATUS=1" >> "$BASH_ENV"
           working_directory: packages/contracts-bedrock
       - run:
-          name: gas snapshot
+          name: gas lock
           command: |
             forge --version
-            pnpm gas-snapshot --check || echo "export GAS_SNAPSHOT_STATUS=1" >> "$BASH_ENV"
+            pnpm gas-lock --check || echo "export GAS_LOCK_STATUS=1" >> "$BASH_ENV"
```

Comment:
this kind of breaks the standard `gas-snapshot` terminology, so I'd be OK with reverting this.

Reply:
Not opinionated here tbh. Both mean the same thing in my head 😄